eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md

```diff
@@ -0,0 +1,44 @@
+# EiffelConfidenceLevelModifiedEvent
+The EiffelConfidenceLevelModifiedEvent declares that an entity has achieved (or failed to achieve) a certain level of confidence, or in a broader sense to tag it. This is particularly useful for promoting various engineering artifacts, such as product revisions, through the continuous integration and delivery pipelin.
```

Comment:
The last part of the sentence "or in a broader sense to tag it" could be improved.

Also a typo at the end of the paragraph: pipelin -> pipeline